packages/expo-video/src/index.ts

```diff
@@ -2,4 +2,4 @@ import Video from './NativeVideoModule';
 
 export { VideoView, useVideoPlayer, isPictureInPictureSupported } from './VideoView';
 export { Video };
-export { VideoSource } from './VideoView.types';
+export { VideoSource, VideoPlayerEvents } from './VideoView.types';
```

Comment:
It might be good to move some player-specific types to another file and use `VideoView.types` only for view-specific types like props. No need to do that in this PR.